rts/lra-examples/cdi-participant/pom.xml

```diff
@@ -7,18 +7,19 @@
     <version>5.9.7.Final-SNAPSHOT</version>
     <artifactId>examples-microprofile-lra-cdi-participant</artifactId>
 
-    <name>WildFly Thorntail Examples: Microprofile LRA CDI Participant</name>
-    <description>WildFly Thorntail Examples: Microprofile LRA CDI participant using annotations</description>
+    <name>MicroProfile Examples: LRA CDI Participant</name>
+    <description>MicroProfile Examples: LRA CDI participant using annotations</description>
 
     <packaging>war</packaging>
 
     <properties>
         <maven.test.skip>true</maven.test.skip>
-        <swarm.debug.port>5005</swarm.debug.port>
+        <thorntail.debug.port>5005</thorntail.debug.port>
         <failOnMissingWebXml>false</failOnMissingWebXml>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <version.wildfly-swarm>2017.8.1</version.wildfly-swarm>
-        <version.microprofile.lra>1.0-20190122.071206-242</version.microprofile.lra>
+        <version.microprofile.lra>1.0-20190812.092710-499</version.microprofile.lra>
```

Comment:
later we use 1.0-RC1. We should stay consistent and either use RC version or constantly updated snapshots.

Reply:
+1